src/utils/message.rs

```diff
@@ -2,17 +2,39 @@ use fvm_ipld_encoding::RawBytes;
 use fvm_shared::message::Message;
 use fvm_shared::{address::Address, econ::TokenAmount, METHOD_SEND};
 
+/// Creates a new transfer message with default values for gas and sequence.
 pub fn message_transfer(from: Address, to: Address, value: TokenAmount) -> Message {
+    message_transfer_custom(
+        from,
+        to,
+        value,
+        0,
+        TokenAmount::from_atto(0),
+        TokenAmount::from_atto(0),
+        0,
+    )
+}
+
+/// Creates a new transfer message with specified values for gas and sequence.
+pub fn message_transfer_custom(
```

Comment:
```suggestion
pub fn message_transfer_native(
```
I think we should make a distinction between native FIL messages and Ethereum ones.

Reply:
done